packages/network-of-terms-catalog/catalog/datasets/goudatijdmachine-straten.jsonld

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://www.goudatijdmachine.nl/id/straten",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Goudse straten"
+    }
+  ],
+  "creator": [
+    {
+      "@id": "https://www.goudatijdmachine.nl",
+      "@type": "http://schema.org/Organization",
+      "name": "Gouda Tijdmachine",
+      "alternateName": "GTM"
+    }
+  ],
+  "url": [
+    "https://n2t.net/ark:/60537/biWGGg"
```

Comment:
Again, this should be the URI _prefix_. This is not the prefix, looking at a random street https://n2t.net/ark:/60537/b87jJw.